release-bintray.gradle

```diff
@@ -67,4 +67,4 @@ ext {
 apply from: rootProject.file('jcenter/maven-install.gradle')
 
 // Publish on Bintray.
-apply from: rootProject.file('jcenter/bintray.gradle')
```

Comment:
Maybe no change is needed here.

Reply:
The change is actually adding a new line to the end of the file. POSIX 3.403 standard requires each line to have a newline character
"A line consisting solely of zero or more characters terminated by a <newline>. A file that contains characters organized into zero or more lines."

So it is actually good to conform the standard and always have a newline at the end of the file.

This is from my git diff
```--- a/release-bintray.gradle
+++ b/release-bintray.gradle
@@ -56,7 +56,7 @@ ext {
   siteUrl = 'https://github.com/linkedin/photon-ml'
   gitUrl = 'https://github.com/linkedin/photon-ml.git'

-  libraryVersion = '20.0.2'
+  libraryVersion = '20.1.0-SNAPSHOT'

   licenseName = 'Apache-2.0'
   licenseUrl = 'http://www.apache.org/licenses/LICENSE-2.0'
@@ -67,4 +67,4 @@ ext {
 apply from: rootProject.file('jcenter/maven-install.gradle')

 // Publish on Bintray.
-apply from: rootProject.file('jcenter/bintray.gradle')
\ No newline at end of file
+apply from: rootProject.file('jcenter/bintray.gradle')```

Reply:
I didn't notice that it was adding a new line. Thanks.